package.json

```diff
@@ -8,23 +8,25 @@
   "description": "Tiny, type-safe wrapper around MSW for type inference from OpenAPI schemas.",
   "repository": "github:christoph-fricke/openapi-msw",
   "files": [
+    "cjs",
     "dist",
     "CHANGELOG.md",
     "README.md",
     "LICENSE"
   ],
   "sideEffects": false,
   "types": "./dist/exports/main.d.ts",
-  "module": "./dist/exports/main.js",
+  "main": "./dist/exports/main.js",
   "exports": {
     ".": {
       "types": "./dist/exports/main.d.ts",
-      "import": "./dist/exports/main.js"
+      "import": "./dist/exports/main.js",
+      "require": "./cjs/exports/main.js"
     }
   },
   "scripts": {
     "prepare": "husky",
-    "build": "rimraf ./dist && tsc -p tsconfig.build.json",
+    "build": "rimraf ./dist ./cjs && tsc -p tsconfig.build.json && tsc -p tsconfig.cjs.json",
```

Comment:
This is really great thank you, I appreciate the extra hassle to support CJS. For this to work with NodeJS it just needs a small change to add a nested package.json in the `cjs` dir so that NodeJS interprets files in this folder as cjs. Since the main package is "type": "module" it will treat all .js files as esm otherwise. Details: https://transang.me/how-to-publish-a-hybrid-npm-package/#step-4-important-make-the-es-entry-work

Add nested `/cjs/package.json`
```json
{
  "type": "commonjs"
}
```

This line can then be changed to not delete the nested package.json on rebuilds.
```suggestion
    "build": "rimraf ./dist ./cjs/**/ && tsc -p tsconfig.build.json && tsc -p tsconfig.cjs.json",
```